syntax = "proto3";

package ErrorCode;

enum EnumErrorCode
{
	//通用错误码
	CODE_SUCCESS = 0;        //成功
	//通信错误码
	CODE_PARAMETER_NULL = 0x0001; //参数为空
	CODE_PARAMETER_INVALID = 0x0002;  //参数不合法
    CODE_INVALID_SITE_ID = 0x0003;  //不合法的SITEID
    CODE_INVALID_SITE_NAME = 0x0004; //不合法的SITENAME
    CODE_WAIT_RESP_TIMEOUT = 0x0005; //等待response超时
	CODE_SYSTEM_ERROR = 0x0006;
	
	//登录错误
	CODE_ACOCUNT_EMPTY = 0x0007;
	CODE_LOGIN_ERROR = 0x0008;
	
	CODE_CHECK_GAME_TOKEN_FAILED = 1000;
	CODE_INVALID_ROOMID = 1001;
};


